src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -40,9 +40,6 @@
                         iconName: contentType.identifier,
                         contentTypeName: contentType.name
```

Comment:
I think it should be now 
```
contentTypeName: contentType.name,
form_location_bookmark: form_location_bookmark
```
as this variable is no longer passed into `location_bookmark.html.twig` right now

Reply:
I have tried this option and the issue persists.